### ISSUE TITLE
Icon Request: world-anvil

### ISSUE THREAD

**Issue by iamromeo** [FortAwesome/Font-Awesome#16601]
**What can it be used for?**
Brand icon for sharing and linking to World Anvil world profiles for users and work. 

**Example image (optional)**
![WA-monogram-black](https://user-images.githubusercontent.com/660175/81419148-b7c2f680-9145-11ea-9b0a-aa9a618631c1.png)

**Icon request checklist**
- [ x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [ x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [ x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x ] I have included at least one example image if it could be helpful (optional)
- [x ] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by Tillerz** [comment]
Need it!

**Comment by SigmaXD** [comment]
Awesome! This is great!!

**Comment by Helvianir** [comment]
It would be very useful.

**Comment by PatheticBarrel** [comment]
The benefits far outweigh the need for there to even be a vote taken.

World Anvil has, since it's creation, been a source of great creativity. Need I say more? I think not.

**Comment by mctribble** [comment]
Would be much appreciated by the WA community

**Comment by melior64** [comment]
Yes please! I use icons for all my communities and only World Anvil is missing

**Comment by iamromeo** [comment]
Categories:
gaming-tabletop / fab

**Comment by KahunaTheElder** [comment]
WA has accelerated my work. This will add to that.  Please add.

**Comment by RandMihkel** [comment]
This would be very valuable.

**Comment by KatAKlettke** [comment]
this would be a great addition!

**Comment by cslontok** [comment]
Would love to have this.

**Comment by Hanhula** [comment]
Would appreciate this!

**Comment by IndustrialScribe** [comment]
I need this :)

**Comment by geoffwasham** [comment]
Please make the WA logo available in FontAwesome! I need it!

**Comment by tagliala** [comment]
Hi everyone, a lot of enthusiasm here!

Please avoid comments and use the upvote function 👍

**Comment by vinceblake** [comment]
It's unlikely that this will cure COVID19, but all we know for sure is that the virus came from a world without this icon. I don't think we can afford to keep living in that world. Do the right thing.

**Comment by tagliala** [comment]
Hi @iamromeo 

I think this issue is being shared somewhere. Could you please ask to avoid comments and use the 👍 reaction instead?

They count the same in our leaderboard

Thanks!

**Comment by iamromeo** [comment]
Yes, will find and will do!

**Comment by meins111** [comment]
The name says it all: World Anvil.
The place were worlds are born from nothing!
Yes, please!

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No worries.
* This brand has gone the way of the dodo.
